src/test/java/graphql/annotations/GraphQLIterableAndArrayTest.java

```diff
@@ -103,8 +105,7 @@ public void init() {
 
     @Test
     public void queryWithArray() {
```

Comment:
I know u probably didn't write those tests..
Maybe change the names according to the standard?